components/admin_console/generated_setting.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import crypto from 'crypto';
+import crypto from 'crypto-browserify';
```

Comment:
Previous versions of `webpack` automatically bundled some Node libraries like `crypto`, but now we have to do it ourselves.

Reply:
Never mind that. We're now using `crypto-browserify` implicitly since that's required for the type definitions to work.